### PR TITLE
feat: createMany integration

### PR DESCRIPTION
**Pull request by timsuchanek**

### CODE REVIEW

**Comment by williamluke4** [comment]
This needs to be merged to unblock @Jolg42. Tests and putting createMany behind the flag are yet to be done. These will be added in a sperate PR

File: src/packages/client/package.json [hunk 1]
```diff
@@ -126,7 +126,7 @@
     }
   },
   "dependencies": {
-    "@prisma/engines-version": "2.16.0-23.0912dc6a337ad2067160deccb6183a43f25928ef"
+    "@prisma/engines-version": "2.16.0-24.010a061820516f9f59b5124ae9d5b8531b19bec9"
```
Comment (Jolg42) [inline src/packages/client/package.json:129]: It's a bit weird it's on 24 and not 25 here

File: src/packages/client/src/generation/utils.ts [hunk 2]
```diff
@@ -138,6 +140,8 @@ export function getModelArgName(
     case DMMF.ModelAction.count:
       return `${modelName}CountArgs`
   }
+  console.log(DMMF.ModelAction.createMany);
+  return 'wtf'
```
Comment (Jolg42) [inline src/packages/client/src/generation/utils.ts:144]: 😄

**Review by Jolg42 — APPROVED** [review]
Looks good (see comments)

File: src/packages/client/src/generation/utils.ts (hunk 2 above)
Comment (williamluke4) [reply src/packages/client/src/generation/utils.ts]: :facepalm:

**Pull request merged**